policyengine_us/tests/policy/baseline/gov/states/nj/tax/income/main/nj_income_tax_before_refundable_credits.yaml

```
@@ -0,0 +1,58 @@
+- name: Single filer, taxable income is $25k in 2022 then tax is $367.50.
+  period: 2022
+  input:
+    nj_taxable_income: 25_000
+    filing_status: SINGLE
+    state_code: NJ
+  output:
+    # .014 * 20,000 + .0175 * (25,000-20,000) = 367.50
+    nj_income_tax_before_credits: 367.5
```

Just a typo, all of the output names in the test need to be changed
```suggestion
    nj_income_tax_before_refundable_credits: 367.5
```